OpenRA.Mods.Common/Lint/CheckTranslationReference.cs

```diff
@@ -103,9 +103,6 @@ void ILintPass.Run(Action<string> emitError, Action<string> emitWarning, ModData
 			{
 				foreach (var fieldInfo in modType.GetFields(Binding).Where(m => Utility.HasAttribute<TranslationReferenceAttribute>(m)))
 				{
-					if (fieldInfo.FieldType != typeof(string))
```

Comment:
Won't this break when getting the string value of the field below?

Reply:
I found out that this gets never triggered.

Reply:
Wait, I forgot. This is required. Put it in a separate commit to make it clearer.

Reply:
This will cause a crash if we have a field that is not `IsInitOnly`. If we want to support dictionaries for such fields (I assume those are all chrome related) then this should be implemented to get all the keys from the dictionary properly. If this should not be supported the check needs to be added back below the `IsInitOnly` check and emit an error there.